augly/video/functional.py

```diff
@@ -44,8 +45,24 @@ def add_noise(
     """
     func_kwargs = helpers.get_func_kwargs(metadata, locals(), video_path)
 
-    noise_aug = af.VideoAugmenterByNoise(level)
```

Comment:
Same comment as on your other diff: if you're no longer using these classes `VideoAugmenterByNoise` etc, please delete those files & the references to them in `video/augmenters/__init__.py` & anywhere else :)

Reply:
Will do, also there's a lot of repetition of the instantiation of a writer, and then executing the ffmpeg command, and then closing the writer. I think we should try to keep the codebase dry instead of wet and maybe declare the ffmpeg command in the function and then pass it into a generic function which does what is repeated.